tests/tests.py

```diff
@@ -91,6 +91,11 @@ def test_subblock_no_parent(self):
         result = render_block_to_string("test_sub.html", "first")
         self.assertEqual(result, "\nbar\n")
 
+    @override_settings(DEBUG=True)
+    def test_exceptions_in_templates(self):
+        with self.assertRaises(Exception):
```

Comment:
Should we use a custom execution class to ensure it is the expected exception that we're throwing?